strapi/Dockerfile

```diff
@@ -1,57 +1,66 @@
-# development
-FROM node:16 AS dev
-
-RUN apt-get update && apt-get install -y git \
-    vim \
-    libvips-dev
-
-ARG NODE_ENV=development
-ENV NODE_ENV=${NODE_ENV}
-
-WORKDIR /home/node/app
-COPY package*.json ./
-COPY yarn.lock ./
+FROM node:16-alpine AS build-base
+RUN apk update \
+ && apk add --no-cache build-base \
+                       gcc \
+                       autoconf \
+                       automake \
+                       zlib-dev \
+                       libpng-dev \
+                       vips-dev > /dev/null 2>&1
```

Comment:
Why you need this `> /dev/null 2>&1`?

Reply:
yes, this was a mistake and was left over as I copied it from the (un)official strapi documentation of package needed to build